webapp/lib/controller.dart

```diff
@@ -82,29 +106,53 @@ model.User signedInUser;
 
 Map<String, Timer> projectTimers = {};
 
+StreamSubscription needsReplyMetricsSubscription;
+List<StreamSubscription> driverMetricsSubscriptions = [];
+
 void init() async {
   view.init();
   await platform.init();
 }
 
 void initUI() {
   needsReplyDataList = [];
+  driversDataMap = {};
   systemEventsDataMap = {};
   systemMetricsDataList = [];
   dirSizeMetricsDataList = [];
 
   selectedTab = view.contentView.getChartTypeUrlFilter() ?? ChartType.conversation;
   view.contentView.toogleTabView(selectedTab);
 
-  selectedPeriodFilter = view.contentView.getChartPeriodFilter() ?? ChartPeriodFilters.alltime;
-  view.ChartFiltersView().periodFilterOptions = ChartPeriodFilters.values;
+  selectedPeriodFilter = view.contentView.getChartPeriodUrlFilter() ?? ChartPeriodFilters.days1;
```

Comment:
Is the default period filter now `1 day` instead of `all time`?

Reply:
yeah, I've changed the default to now `1 day`. I think we're heading for Nook Watcher to be an active present monitoring tool rather than historical, and will do the historical data analysis somewhere else, meaning that `all time` graphs are less useful. They also take a long time to load esp. for long-running projects, so we might also want to move to a model where we query firebase only for the most recent data.

Reply:
okay. Makes sense